### PR TITLE
new getting started guide for Rider

### PR DESCRIPTION
**Pull request by alimogithub**
new guide for JetBrains Rider + some changes to existing screenshots

### CODE REVIEW

**Comment by SabotageAndi** [comment]
@alimogithub Could do some small change and push it again? I enabled the check for PRs on this repo now. It makes it easier to review the changes for me if I see them in the final documentation.

**Comment by SabotageAndi** [comment]
What about having here two big images with VS and Rider logo to decide which getting started you want to do?

image

File: index.rst [hunk 1]
```diff
@@ -41,6 +43,29 @@ You can also get help by reaching out to our support team `here <https://go.spec
...
+   gettingstartedrider/Step4r.md
+   gettingstartedrider/Step5r.md
+   gettingstartedrider/Step6r.md
+   gettingstartedrider/Step7r.md
+   gettingstartedrider/Step8r.md
+   gettingstartedrider/Step9r.md
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Additional Resrouces
```
Comment (SabotageAndi) [inline index.rst:66]: Typo ;-)

File: index.rst [hunk 2]
```diff
@@ -27,8 +29,8 @@ You can also get help by reaching out to our support team `here <https://go.spec
 
 .. toctree::
    :maxdepth: 1
-   :caption: Getting Started from Scratch
-   :numbered:
+   :caption: Getting Started-Visual Studio
```
Comment (SabotageAndi) [inline index.rst:32]: Do we need "Getting Started" here at the beginning?
Isn't it clear that this is the getting started? Looks at the moment a little bit strange and is hard to read.

**Review by SabotageAndi — CHANGES_REQUESTED** [review]

File: index.rst (hunk 2 above)
Comment (alimogithub) [reply index.rst:32]: yup, removed

**Review by SabotageAndi — APPROVED** [review]

**Pull request merged**